### DISCUSSION TITLE
Install 3.6.2 headless server?

### DISCUSSION THREAD

**Discussion by chrisrimple** [jamulussoftware/jamulus#1645] [Technical Support & Help]
I want to try reverting my headless 3.7.0 server to 3.6.2 to see if it solves sound dropout issues that I'm experiencing.  I previously used @sthenos's [install script](https://www.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/), which includes the reference:

https://api.github.com/repos/jamulussoftware/jamulus/releases/latest

That downloads https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_ubuntu_amd64.deb.  What would be the API path for 3.6.2?  I'm planning to edit the script to get 3.6.2 instead, after destroying/recreating the Ubuntu server.

Thanks!

**Comment by ghost** [comment]
https://sourceforge.net/projects/llcon/files/
Looks like there is only source code (which is available on github also).
You will have to compile the source, or remember where you downloaded the executable.
BTW the Jamulus amd64.deb is probably packaged with your distro.

**Reply by chrisrimple** [reply]
Thanks.  Looks like the URL provided does not include a .deb file.

**Comment by hoffie** [comment]
Some weeks ago there had been a similar request. As a response, I did a quick, inofficial almost-3.6.2 build including Debian packages here: https://github.com/hoffie/jamulus/releases/tag/r3_6_2-hoffie-autobuild ([headless .deb](https://github.com/hoffie/jamulus/releases/download/r3_6_2-hoffie-autobuild/jamulus_headless_3.6.2-dev-hoffie-8af66df_ubuntu_amd64.deb))

It's "almost" 3.6.2 because I had to backport several build-related changes. The actual code (e.g. audio processing, network, UI) is vanilla 3.6.2 though. You can see the changes here:
https://github.com/hoffie/jamulus/commits/r3_6_2-hoffie-autobuild

I just notice that it does contain some unwanted windows/ changes though. :(
Maybe it helps nevertheless. It is completely untested and -- as said -- non-official.
